server/src/test/java/org/opensearch/index/codec/CodecTests.java

```diff
@@ -72,16 +75,37 @@ public void testResolveDefaultCodecs() throws Exception {
 
     public void testDefault() throws Exception {
         Codec codec = createCodecService().codec("default");
-        assertCompressionEquals(Mode.BEST_SPEED, codec);
+        assertDVCompressionEquals(Lucene80DocValuesFormat.Mode.BEST_COMPRESSION, codec);
+        assertStoredFieldsFormatCompressionEquals(Lucene87StoredFieldsFormat.Mode.BEST_COMPRESSION, codec);
```

Comment:
Why do the stored fields now default to `BEST_COMPRESSION`? This is not an appropriate default: it should be `BEST_SPEED`. For DV `BEST_COMPRESSION` is appropriate here...

Reply:
:+1:  Thanks @rmuir! This was an unintentional tab completion miss on my part.

Reply:
OK, the crazy compression options are a personal pet-peeve in lucene, (I added more comments to https://issues.apache.org/jira/browse/LUCENE-9843). Lucene nightly benchmark even accidentally uses wrong settings!